rules/feel.js

```diff
@@ -0,0 +1,66 @@
+const { is } = require('bpmnlint-utils');
+const { lintExpression } = require('@bpmn-io/feel-lint');
+
+const { reportErrors } = require('./utils/reporter');
+const { getPath } = require('@bpmn-io/moddle-utils');
+const { ERROR_TYPES } = require('./utils/error-types');
+
+module.exports = function() {
+  function check(node, reporter) {
+
+    const parentNode = findFlowElement(node);
+    const errors = [];
+
+    if (!parentNode) {
+      return;
+    }
```

Comment:
Should never happen, but it does: https://github.com/bpmn-io/bpmn-js/issues/1754

Better be safe than blow up 😄